packages/feedback/feedback.scss

```
@@ -0,0 +1,114 @@
+@import "~@fremtind/jkl-core/mixins/_all.scss";
+@import "~@fremtind/jkl-core/variables/_all.scss";
+@import "~@fremtind/jkl-core/_functions.scss";
+
+$feedback-icons: (#ff7059, #f2994a, #ffc880, #50e68c, #27ae60);
```

```suggestion
$feedback-icon-colors: (#ff7059, #f2994a, #ffc880, #50e68c, #27ae60);
```

Ser egentlig ikke ut som denne brukes noe sted?

---

nope, glemte å fjerne etter en forenkling